test.runsettings

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+   <MSTest>
+      <!-- Specify timeout in milliseconds. A valid value should be greater than 0 -->
+      <TestTimeout>300000</TestTimeout>
```

Comment:
This is per test, correct?

Reply:
It may be a bit aggressive. A timeout like this should only be used to stop stalling tests, so long running tests may hit this threshold if we keep it at 5 minutes

Reply:
yes

Reply:
I agree, I am running the entire test suite to see how many tests get timeout out with this value

Reply:
Okay, I trust you will come up with an appropriate value here then, so I'll sign off